torchaudio/datasets/musdb_hq.py

```diff
@@ -0,0 +1,104 @@
+import os
+from pathlib import Path
+from typing import Tuple, Union, List, Optional
+
+import torch
+import torchaudio
+from torch.utils.data import Dataset
+
+_URL = "https://zenodo.org/record/3338373/files/musdb18hq.zip"
+_EXT = ".wav"
+_SAMPLE_RATE = 44100
+
+
+class MUSDB_HQ(Dataset):
+    """Create *MUSDB_HQ Dataset* [:footcite:`MUSDB18HQ`] Dataset
+
+    Args:
+        root (str or Path): Root directory where the dataset's top level directory is found
+        subset (str): Subset of the dataset to use. Options: [``"train"``, ``"test"``].
+        sources (List[str] or None, optional): Sources extract data from.
+            Options: [``bass``, ``drums``, ``other``, ``mixture``, ``vocals``].
```

Comment:
@nateanl wdyt of this wording, any additional suggestion?

```suggestion
        sources (List[str] or None, optional): Sources to extract data from.
            List can contain the following options: [``"bass"``, ``"drums"``, ``"other"``, ``"mixture"``, ``"vocals"``].
```

Reply:
ohh I understand. The sources here mean it can be any combination of these options? 
For example:
["bass", "drums"] is one option, ["mixture", "vocals"] is also an option?